CoffeeChat/Controller/LoginViewController.swift

```diff
@@ -0,0 +1,83 @@
+//
+//  LoginViewController.swift
+//  CoffeeChat
+//
+//  Created by HAIYING WENG on 1/29/20.
+//  Copyright © 2020 cuappdev. All rights reserved.
+//
+
+import GoogleSignIn
+import SnapKit
+import UIKit
+
+class LoginViewController: UIViewController {
+
+    private let loginButton = GIDSignInButton()
+    private let logoImageView = UIImageView()
+
+    private let logoSize: CGFloat = 150
```

Comment:
lets just use the value directly instead of creating a variable since you only use it once

Reply:
oh i put it there cuz `logoSize` also used for `cornerRadius`